exaudfclient/docs/script_options_requirments.md

```diff
@@ -0,0 +1,363 @@
+
+Tags: V1
+
+### Script Option Unknown Options Behavior V2
+`req~script-option-unknown-options-behvaior-v2~1`
+
+The parser handler must detect unknown script options and throw an exception if such an options is found. 
+
+Needs: dsn
+
+Covers:
+- `feat~general-script-options-parsing~1`
+- `feat~java-specific-script-options~1`
+
+Tags: V2
+
+### Escape Sequence Script Options Parsing
+`req~escape-sequence-script-options-parsing~1`
+
+The following rules for escape sequences at any place within a script optionValue are to be applied:
+- '\n' => <line feed> character
+- '\r' => <carriage return> character
+- '\;' => ';' character
+
+Needs: dsn
+
+Tags: V2
+
+Covers:
+- `feat~general-script-options-parsing~1`
+
+
+### Java %scriptclass Option Handling V1
+`req~java-scriptclass-option-handling-v1~1`
+
+The Java parser handler must correctly identify the first `%scriptclass` option and remove only this single instance from the script code. Any further occurrences of `%scriptclass` option shall stay in the source script code.
+The value should be handled according to the [Java specification for identifies](https://docs.oracle.com/javase/specs/jls/se7/html/jls-3.html#jls-3.8).
+
+Needs: dsn
+
+Tags: V1
+
+Covers:
+- `feat~java-specific-script-options~1`
+
+
+### Java %scriptclass Option Handling V2
+`req~java-scriptclass-option-handling-v2~1`
+
+The Java parser handler must correctly identify the first `%scriptclass` option and remove any additional occurrences of this option within the script code.
+The value should be handled according to the [Java specification for identifies](https://docs.oracle.com/javase/specs/jls/se7/html/jls-3.html#jls-3.8).
+
+Needs: dsn
+
+Tags: V2
+
+Covers:
+- `feat~java-specific-script-options~1`
+
+### Java %jar Option Handling Multiple Options
+`req~java-jar-option-handling-multiple-options~1`
+
+The Java parser handler must find multiple `%jar` options. The values are to be interpreted as the Java CLASSPATH: `<file1>:<file2>:...:<filen>`. The Java parser handler shall split the entries by the colon character.
+Compare [OpenJdk implementation](https://github.com/AdoptOpenJDK/openjdk-jdk11/blob/19fb8f93c59dfd791f62d41f332db9e306bc1422/src/java.base/share/classes/jdk/internal/loader/URLClassPath.java#L174) of parsing the classpath.
+
+Covers:
+- `feat~java-specific-script-options~1`
+
+### Java %jar Option Handling V1
+`req~java-jar-option-handling-v1~1`
+
+The Java parser handler shall unify duplicated files and order the result of all `%jar` options alphabetically.
+
+Needs: dsn
+
+Tags: V1
+
+Covers:
+- `feat~java-specific-script-options~1`
+
+Depends:
+- `req~java-jar-option-handling-multiple-options~1`
+
+### Java %jar Option Handling V2
+`req~java-jar-option-handling-v2~1`
+
+The Java parser handler must keep duplicates. The order of the entries must not change.
+
+Needs: dsn
+
+Tags: V2
+
+Covers:
+- `feat~java-specific-script-options~1`
+
+Depends:
+- `req~java-jar-option-handling-multiple-options~1`
+
```

Comment:
Jar option in v2 needs to be able to deal with whitespace in the jar paths. This requirement is missing.